pkgs/development/libraries/criterion/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit, cmake, pkg-config, boxfort, libcsptr, dyncall, nanomsg }:
+
+stdenv.mkDerivation rec {
+  pname = "criterion";
+  version = "2.3.3";
+
+  src = fetchgit
+    { url = "git://github.com/Snaipe/Criterion";
+      rev = "v${version}";
+      fetchSubmodules = true;
+      sha256 = "0y1ay8is54k3y82vagdy0jsa3nfkczpvnqfcjm5n9iarayaxaq8p";
+    };
+
+  buildInputs =
```

Comment:
`cmake` and `pkg-config` should be in `nativeBuildInputs` instead of `buildInputs`.

Reply:
Fixed